docs/team/a1021492980.md

```diff
@@ -1,6 +1,99 @@
+
+
+* Code Contributed:
+    * Out of around 5100 lines of functional code of the whole project, 1334 lines of functional code were contributed by
+  me
+  [RepoSense link](https://nus-cs2113-ay2122s2.github.io/tp-dashboard/?search=a1021492980&breakdown=true&sort=groupTitle&sortWithin=title&since=2022-02-18&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other).
+
+
+* Contributions to the UG:
+  * Added documentation for the features `search /exercise`, `search /workout`, `search /plan` and `search /all`.
+  * Added documentation for the feature `exercise /list`.
+  * Added documentation for the feature `help`.
+
+
+* Contributions to the DG:
+  * Added documentation for the features `search /exercise`, `search /workout`, `search /plan` and `search /all`.
+  * Added documentation for the feature `exercise /list`.
+  * Added sequence diagrams for `exercise /list`, `search /exercise`, `search /workout`, `search /plan` and `search /all`.
+  * Added class diagram for `SearchCommand` and `ExerciseCommand`.
+  * Added design consideration for inconsistent data between resource files under  `File Management`.
+  * Added `Exercise-related feautures` and `Search-related features` under `design` section.
+
+
+* Community:
+  * Maintaining the issue tracker:
+    * Creat issues to be done by me, when creating PRs, link them to the corresponding issues.
+    
+  * Review contributions (Including but not limited to) : 
+    * [review 1](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/17#discussion_r820104333)
+    * [review 2](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/22#discussion_r822512048)
+    * [review 3](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/105#discussion_r831350578)
+    * [review 4](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/105#discussion_r831338031)
+    * [review 5](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/105#discussion_r831344059)
+  * General bugs fixing (Including but not limited to) :
+    * Fix bugs related to file loading. [PR Link](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/218#issue-1190273926)
+    * Fix bugs related to data cascading due to deletion of relevant workouts/plans. [PR Link](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/218#issue-1190273926)
+    * Routine bug fixes. [PR Link](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/216#issue-1190069253), [PR Link](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/73#issue-1170675218)
+
+
+* Contributions beyond the project team:
+  * Bugs reported in other team's products:
+    * [Bug reported 1](https://github.com/a1021492980/ped/issues/4#issue-1189515964)
+    * [Bug reported 2](https://github.com/a1021492980/ped/issues/3#issue-1189500495)
+    * [Bug reported 3](https://github.com/a1021492980/ped/issues/2#issue-1189492498)
+    * [Bug reported 4](https://github.com/a1021492980/ped/issues/1#issue-1189476366)
```

Comment:
Perhaps you might want to add a bit of description so reader can understand it better.